packages/eslint-plugin/src/rules/explicit-function-return-type.ts

```diff
@@ -69,6 +70,11 @@ export default util.createRule<Options, MessageIds>({
             },
             type: 'array',
           },
+          allowIIFEs: {
+            description:
+              'Whether to ignore functions immediately invoked (IIFEs).',
```

Comment:
Nit: IIFE is the technical term and is more precise. I'd say let's stick with it?

```suggestion
              'Whether to ignore immediately invoked function expressions (IIFEs).',
```